db/compaction_picker_universal.cc

```diff
@@ -342,8 +342,6 @@ Compaction* UniversalCompactionPicker::PickCompaction(
       assert(f->smallest_seqno <= f->largest_seqno);
```

Comment:
For this part, I also think it is unnecessary (the code updates the seqno boundary should be in version_edit.h). But just keep it if I dont know about it :)